app/screens/home/channel_list/servers/servers_list/server_item/options/index.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {Animated, StyleSheet, View} from 'react-native';
+
+import {useTheme} from '@context/theme';
+import {changeOpacity} from '@utils/theme';
+
+import Option, {OPTION_SIZE} from './option';
+
+import type ServersModel from '@typings/database/models/app/servers';
+
+type Props = {
+    progress: Animated.AnimatedInterpolation;
+    server: ServersModel;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: 'row',
+        marginLeft: 12,
+        width: OPTION_SIZE * 3,
+    },
+    left: {borderTopLeftRadius: 8, borderBottomLeftRadius: 8},
+    right: {borderTopRightRadius: 8, borderBottomRightRadius: 8},
+});
+
+const ServerOptions = ({progress, server}: Props) => {
+    const intl = useIntl();
+    const theme = useTheme();
+
+    const onEdit = useCallback(() => {
+        // eslint-disable-next-line no-console
```

Comment:
I guess these (as many others in this PR) are to be handled by the next PRs, right? I assume they do not need any kind of ticket.

Reply:
no need for ticket as that is part of the current ticket, just splitting the PR's to make it a little bit smaller and faster to review